app-modules/case-management/src/Filament/Resources/CaseTypeResource/Pages/EditCaseTypeAssignments.php

```diff
@@ -69,7 +68,7 @@ public function getRelationManagers(): array
 
     public static function canAccess(array $parameters = []): bool
     {
-        return AssignmentsFeature::active() && parent::canAccess($parameters);
+        return parent::canAccess($parameters);
     }
```

Comment:
Because all you are doing here now is just returning the parent, the entire method override can just be removed.